index.html

```diff
@@ -1,25 +1,43 @@
-<doctype! html>
+<!DOCTYPE html>
 <html>
+
 <head>
-  <title>Seattle M's: Playoffs?</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Steven's Guessing Game</title>
+  <link rel="stylesheet" href="">
 </head>
+
   <body>
     <h1>Will the Seattle Mariners make the playoffs?</h1>
-      <p><img src="mariners1.jpg" width=300px height 75px></p>
-        <p>The idea here is to guess the amount of wins the Seattle Mariners will end up with after a short 162-game season </p>
+      <img src="img/mariners1.jpg" width=400px height=225px>
+  <p id="result"></p>
+
+    <script type="text/javascript" src="js/main.js"></script>
```

Comment:
This script tag should be down at the bottom of the html document, right above the closing Body tag. That may be part of the issue you're having with an undefined variable name.